docker-compose-mode-files/docker-compose-df-only.yaml

```
@@ -2,11 +2,12 @@ version: "3.3"
 services:
  # CB-Dragonfly
   cb-dragonfly:
-    image: cloudbaristaorg/cb-dragonfly:0.3.8
+    image: cloudbaristaorg/cb-dragonfly:dev
     container_name: cb-dragonfly
     volumes:
-      - ./conf/cb-dragonfly:/go/src/github.com/cloud-barista/cb-dragonfly/conf
-      - ./data/cb-dragonfly/log:/go/src/github.com/cloud-barista/cb-dragonfly/log
+      - ./conf/:/go/src/github.com/cloud-barista/cb-dragonfly/conf/
+      - ./meta_db/:/go/src/github.com/cloud-barista/cb-dragonfly/meta_db/
+      - ./log/:/go/src/github.com/cloud-barista/cb-dragonfly/log/
```

cb-operator 디렉토리 구조를 고려하여, 이 부분은 바꾸지 않고 원래대로 두는 것이 어떨까요? 😊

```suggestion
      - ./conf/cb-dragonfly:/go/src/github.com/cloud-barista/cb-dragonfly/conf
      - ./data/cb-dragonfly/log:/go/src/github.com/cloud-barista/cb-dragonfly/log
```